analytical_engine/core/io/property_parser.h

```diff
@@ -133,6 +133,7 @@ struct Graph {
   bool retain_oid = true;
   bool compact_edges = false;
   bool use_perfect_hash = false;
+  int extend_type = 0; // 0 means no need to extend label data
```

Comment:
Explain this with more details please.  the available options are 0/1, 0 stands for xx 1 stands for xx. And "extend label data" is confusing, you have to make a person who doesn't know your work before can understand your comment.

Reply:
Yes, of course. I've ignored this. Thanks a lot for the tips.